packages/common/container-definitions/src/deltas.ts

```diff
@@ -61,21 +61,94 @@ export interface IDeltaSender {
     flush(): void;
 }
 
-/** Events emitted by the Delta Manager */
+/**
+ * Events emitted by {@link IDeltaManager}.
+ */
+/* eslint-disable @typescript-eslint/unified-signatures */
 export interface IDeltaManagerEvents extends IEvent {
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `messageBuffer`: TODO
+     */
     (event: "prepareSend", listener: (messageBuffer: any[]) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     */
     (event: "submitOp", listener: (message: IDocumentMessage) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     *
+     * - `processingTime`: TODO
+     */
     (event: "op", listener: (message: ISequencedDocumentMessage, processingTime: number) => void);
+
+    /**
+     * TODO
+     */
     (event: "allSentOpsAckd", listener: () => void);
-    (event: "pong" | "processTime", listener: (latency: number) => void);
+
+    /**
+     * TODO
```

Comment:
I think we'd discourage use of this by external customers.

Reply:
I marked it as `@internal`. I know we don't yet have a great cohesive story around internal-only exports, but it seems like the approach we've taken elsewhere for this. Let me know if you would prefer this be communicated in another way.

Reply:
Still curious what this means semantically, though. I'm not familiar with the "ping"/"pong" nomenclature in this context.

Reply:
afaik it's used for establishing ping performance telemetry (send a ping, get a pong).  I have not messed with it personally though, so not sure how it really shows up currently.

Reply:
Do you happen to know who might be able to lend further context on this?

Reply:
is it used anywhere? it should have only ever been used internally, i would at least mark it internal, but probably deprecate it, as it may not even work

Reply:
i don't see anywhere in the code where ping or pong is sent, this include server, i think its dead code